notifications_utils/recipients.py

```diff
@@ -381,6 +387,11 @@ def __init__(
                 self.message_too_long = template.is_message_too_long()
             self.message_empty = template.is_message_empty()
 
+            if self.template_type == "letter":
+                self.qr_code_too_long = self._has_qr_code_with_too_much_data()
+            else:
+                self.qr_code_too_long = False
```

Comment:
The `if` statement is redundant since you perform the same check inside the method
```suggestion
            self.qr_code_too_long = self._has_qr_code_with_too_much_data()
```

Reply:
Good spot yeah